tests/core/kernel-and-base-single-reboot-failover/task.yaml

```diff
@@ -4,6 +4,15 @@ summary: Exercises a simultaneous kernel and base refresh with a single reboot
 systems: [ubuntu-core-18-*]
```

Comment:
Is this working now for uc20+, so this can be enabled for these systems? (I see changes for uc20 but not for 22, should be a very minor change to get this running on uc22 too)

Reply:
Still testing to make sure its working on UC20 as well, but I don't know if we neccessarily want to run it on all core versions as it might be a bit overkill.

Reply:
Ok, but then better to run for UC22 instead of UC20 ;)